openquake/risklib/riskinput.py

```diff
@@ -290,14 +287,14 @@ def __init__(self, imt_taxonomies, hazard_by_site, assets_by_site,
             self.weight += len(assets)
         self.taxonomies = sorted(taxonomies)
         self.tags = None  # for API compatibility with RiskInputFromRuptures
-        self.eps_dict = eps_dict or {}
+        self.eps = {} if eps is None else eps
```

Comment:
Indeed there was a confusion here. The class `RiskInput` wants an epsilon dictionary and not a matrix; it is the class `RiskInputFromRuptures` that wants an epsilon matrix. I have updated the docstrings and the name of the parameter accordingly.